Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -165,7 +165,9 @@ define([
                 that.state = Cesium3DTileContentState.FAILED;
                 that._readyPromise.reject(error);
             });
+            return true;
         }
+        return false;
```

Comment:
Since we prefer early return, flip the above `if(defined(promise))` check and return false immediately to make the code cleaner.